internal/servicedeployer/terraform_test.go

```diff
@@ -138,15 +161,14 @@ func TestAddTerraformOutputs(t *testing.T) {
 	for _, tc := range testCases {
 
 		t.Run(tc.testName, func(t *testing.T) {
-			tc.svcInfo.CustomProperties = make(map[string]interface{})
```

Comment:
As now it is used as reference, there is no need to create in advance the map.

Reply:
:+1: this was indeed cheating the test.